### ISSUE TITLE
are yolact-resnet50-fpn-pytorch default scale values wrong in the yml?

### ISSUE THREAD

**Issue by sonebu** [openvinotoolkit/open_model_zoo#1968]
The model.yml for the yolact-resnet50-fpn-pytorch gives mean: [123.675,116.28,103.53] and **scale: [51.525,50.4,50.625]** [here](https://github.com/openvinotoolkit/open_model_zoo/blob/338630987b403a6981d03ab6d04c2d5ad367793a/models/public/yolact-resnet50-fpn-pytorch/model.yml#L100). I'm assuming this is for RGB.

The scale values here don't match with 

1) those provided in the description for the model in the open model zoo [here](https://github.com/openvinotoolkit/open_model_zoo/blob/master/models/public/yolact-resnet50-fpn-pytorch/yolact-resnet50-fpn-pytorch.md), which gives mean: [123.675,116.28,103.53] and **scale: [58.395,57.12,57.375]**. It's stated that these are for RGB.

2) The mean/std values hardcoded in the original repository [here](https://github.com/dbolya/yolact/blob/master/data/config.py#L28), which are MEANS = (103.94, 116.78, 123.68) and **STD = (57.38, 57.12, 58.40)** for BGR. 

(mean values are a bit off too by the way, but they are much closer compared to the scale values ofcourse). Are the scale values in model.yml wrong or am I maybe missing something?

**Comment by eaidova** [comment]
@sonebu thank you for your note, yes, looks like in model.yml we have incorrect values.
Model assumes RGB channels order, in preprocessing the author normalize image, then reverts channels order (that is why they written in BGR), we prefer to do it in opposite order. Also like it comes from backbone, we use standard torchvision normalization values instead provided here. (I do not remember why I select these values instead of provided in the config, but more likely model gives slightly better accuracy with them)

**Comment by sonebu** [comment]
I just generated models with both and tested them on the COCO validation set (5k images, 2017) using the openvino accuracy_checker (ms_coco_mask_rcnn_short_80_classes_without_background) :

[51.525,50.4,50.625] scale values version gives:
AP@masks: 22.78%
AP@boxes: 30.02%

[58.395,57.12,57.375] scale values version gives:
AP@masks: 22.82%
AP@boxes: 30.00%

so I guess there's not much difference after all. Thanks for your help.

**Comment by eaidova** [comment]
@sonebu thank you again, we update our configuration
